.github/workflows/pull_request.yml

```
@@ -38,3 +38,12 @@ jobs:
         env:
           IMG_RESULT: cache
```

```suggestion
          IMG_RESULT: load
```

Can you change it to load. cache means the result will be available only in the buildkit cache. It needs to be available to the local docker daemon for trivy to check it.

@prakhargurunani PTAL.